### PR TITLE
Visitor can read a specific article MOBILE

### PR DESCRIPTION
**Pull request by ViktorHek**
[Visitor can read a specific article MOBILE](https://www.pivotaltracker.com/story/show/176444831)
### User Story
```
As a Registered User, 
In order to read a specific article, 
I would like to be able to click on an article and have it displayed.
```
## Changes proposed in this pull request:
* adding single article component 
* adding current article action in rootReduser
*  adding TouchableOpacity to navigate to single article
## screen shot 

Skärmavbild 2021-01-15 kl  21 36 23

### CODE REVIEW

File: components/SingleArticle.jsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    color: "black",
+    fontSize: 10,
+  },
+  image: {
+    height: 250,
+    width: Dimensions.get("window").width,
+  },
+});
+
+// const SingleArticle = ({ navigation, article }) => {
```
Comment (salindholm) [inline components/SingleArticle.jsx]: Please remove all these comments

File: App.jsx [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+import HomeScreen from "./components/HomeScreen";
+import SingleArticle from "./components/SingleArticle";
 
-
-const Stack = createStackNavigator()
+const Stack = createStackNavigator();
 
 const App = () => {
-  const { appHeader } = useSelector(state => state)
+  const { appHeader } = useSelector((state) => state);
```
Comment (salindholm) [inline App.jsx]: Please remove the parenthesis around (state) for readability

File: components/SingleArticle.jsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
+import React, { useEffect } from "react";
+import { StyleSheet, Text, View, Image, Dimensions } from "react-native";
+import { useSelector } from "react-redux";
+import ArticleServices from "../modules/ArticleServices";
+
+const SingleArticle = (props) => {
+  const { id } = props.route.params.article;
+  const { currentArticle } = useSelector((state) => state);
```
Comment (salindholm) [inline components/SingleArticle.jsx]: Same here, remove the parenthesis around (state)

File: state/reducers/rootReducer.js [hunk 4]
```diff
@@ -1,11 +1,17 @@
+import { action } from "commander";
```
Comment (salindholm) [inline state/reducers/rootReducer.js]: Why are you importing this here?

**Review by salindholm — CHANGES_REQUESTED** [review]
Good job, left some comments!

File: components/SingleArticle.jsx (hunk 1 above)
Comment (ViktorHek) [reply components/SingleArticle.jsx]: done! :)

File: App.jsx (hunk 2 above)
Comment (ViktorHek) [reply App.jsx]: fixed :)

File: components/SingleArticle.jsx (hunk 3 above)
Comment (ViktorHek) [reply components/SingleArticle.jsx]: done :)

File: state/reducers/rootReducer.js (hunk 4 above)
Comment (ViktorHek) [reply state/reducers/rootReducer.js]: honestly... no ide :O. but I have removed it

**Pull request merged**